persist/fs/files.go

```diff
@@ -157,13 +274,27 @@ func TimeFromFileName(fname string) (time.Time, error) {
 	return t, err
 }
 
-// TimeAndIndexFromFileName extracts the block start and index from file name.
-func TimeAndIndexFromFileName(fname string) (time.Time, int, error) {
+// TimeAndIndexFromCommitlogFilename extracts the block start and index from file name for a commitlog.
+func TimeAndIndexFromCommitlogFilename(fname string) (time.Time, int, error) {
+	return timeAndIndexFromFileName(fname, 2)
```

Comment:
nit: use a const decl (within the function or global) instead of the literal. 

Could you ensure there's a test which goes from (time, index) -> filename -> (time, index) to make sure we catch issues if this ever changes in the future.

Reply:
done